test/Sentry.Maui.Device.TestApp/Sentry.Maui.Device.TestApp.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks></TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_ANDROID)' == ''">$(TargetFrameworks);net8.0-android;net9.0-android</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_IOS)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net8.0-ios;net9.0-ios</TargetFrameworks>
+<!--    <TargetFrameworks>$(TargetFrameworks);net8.0-android</TargetFrameworks>-->
+<!--    <TargetFrameworks>$(TargetFrameworks);net8.0-ios</TargetFrameworks>-->
+    <TargetFrameworks>$(TargetFrameworks);net9.0-android</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworks);net9.0-ios</TargetFrameworks>
     <!-- Pin target iOS version so that our tests don't break when new versions of Xcode are released.
       'net8.0-ios' resolves the latest version of the iOS SDK otherwise. -->
-    <TargetPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">18</TargetPlatformVersion>
+
```

Comment:
erm... won't this break our tests when a new version of Xcode is released (per the comment, that hasn't been removed here)?

Reply:
It was moved, not removed.  Line 40 is where it moved to.  It sits with the other versioning values.